modules/install-sno-ibm-z.adoc

```diff
@@ -33,7 +33,7 @@ $ ARCH=<architecture> <1>
 +
 [source,terminal]
 ----
-$ curl -k https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$OCP_VERSION/openshift-client-linux.tar.gz -o oc.tar.gz
+$ curl -k https://mirror.openshift.com/pub/openshift-v4/s390x/clients/ocp/<ocp_version>/openshift-client-linux.tar.gz -o oc.tar.gz
```

Comment:
```suggestion
$ curl -k https://mirror.openshift.com/pub/openshift-v4/${ARCH}/clients/ocp/${OCP_VERSION}/openshift-client-linux.tar.gz -o oc.tar.gz
```

Reply:
Discussed with @smolin-rh that the reason to not use angle brackets in this case is that you can copy the command and run without further changes. Other option would have been to remove step 1 and 2.

Reply:
New changes LGTM